platform_versioned_docs/version-24.1/compute-envs/azure-batch.mdx

```diff
@@ -225,9 +249,9 @@ Nextflow can authenticate to Azure services using a managed identity. This metho
 When you use a manually configured compute environment with a managed identity attached to the Azure Batch Pool, Nextflow can use this managed identity for authentication. However, Platform still needs to use access keys to submit the initial task to Azure Batch to run Nextflow, which will then proceed with the managed identity for subsequent authentication.
```

Comment:
is "managed identity" clear here? I don't see the term referenced before in this doc